src/data/javascript-quiz.ts

```diff
@@ -1058,6 +1058,17 @@ const javascriptQuiz = [
     Link:
       "https://www.freecodecamp.org/news/what-is-javascript-javascript-code-explained-in-plain-english/",
   },
+  {
+    Question: "Which company created JavaScript?",
+    Answer: "Netscape",
+    Distractor1: "IBM",
+    Distractor2: "Xerox",
+    Distractor3: "Google",
+    Explanation:
+      "The first ever JavaScript was created by Brendan Eich at Netscape.",
```

Comment:
we want to avoid using direct quotes from 3rd party resources.

```suggestion
      "Brendan Eich created JavaScript when he was working at Netscape.",
```